packages/twenty-front/src/modules/ui/layout/page/components/PageHeader.tsx

```diff
@@ -27,7 +28,8 @@ const StyledTopBarContainer = styled.div`
   justify-content: space-between;
   min-height: ${PAGE_BAR_MIN_HEIGHT}px;
   padding: ${({ theme }) => theme.spacing(2)};
-  padding-left: 0;
+  padding-left: ${({ isShowFullscreen, theme }) =>
```

Comment:
It feels a bit ugly to have this special condition in header, ideally the header should remain the same everything. Can't we have this in the parent component?